### DISCUSSION TITLE
The detail tap extension is not visible.

### DISCUSSION THREAD

**Discussion by M-SPM** [coder/code-server#3909] [Q&A]
Hello.

I have succeeded in remotely accessing the code server.

But there was one problem.

I can't see detail tap of extension as below I attached picture.

![image](https://user-images.githubusercontent.com/77132725/128443519-e8a0da4a-914a-4972-8e4f-b75d64eeea75.png)

And this problem also occur in git history extension.

How can I solve this??

This is my version.

code-server: v3.11.0
VS Code: v1.57.1
Commit: 4e8cd09ef0412dfc7b148b7639a692e20e4fd6dd
Date: 2021-07-14T20:35:32Z (3 wks ago)
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36

Thank you.

**Comment by jsjoeio** [comment]
Hmm...I'm not able to reproduce. Are there any logs in the console? Do you have an adblocker or something blocking certain websites?

![image](https://user-images.githubusercontent.com/3806031/128547639-fb62dd8c-9c1a-4b03-a6ee-7db44ae057d0.png)

**Reply by M-SPM** [reply]
Thank you for reply~! 

here is my console.
![image](https://user-images.githubusercontent.com/77132725/128605223-44c16616-c140-4735-b686-e9a72a3bec5c.png)

**Reply by jsjoeio** [reply]
Okay nothing unusual there. How about in the browser? Can you check the logs there?
